contrib/hawq-docker/Makefile

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/make all
+
+THIS_MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+TOP_DIR := $(abspath $(dir ${THIS_MAKEFILE_PATH}))
+NDATANODES := 3
+CUR_DATANODE := 1
```

Comment:
I think NDATANODES is the total number of datanodes. And CUR_DATANODE is a iteration placeholder during the loop for start/stop/run/other operations for the hdfs.  

I think a configurable number of datanodes are more reasonable. And you can set 3 as default value.